Accessing-the-database-from-your-frontend.md

```diff
@@ -1,5 +1,9 @@
+# Accessing the database from your front-end

```

Comment:
This article needs to be renamed.

It seems to be part of a series so i would suggest the name of the root article as the root of the new name.